Code/Editor/Lib/Tests/test_Main.cpp

```diff
@@ -43,7 +43,7 @@ AZTEST_EXPORT int AZ_UNIT_TEST_HOOK_NAME(int argc, char** argv)
     // end
     AZ::Test::ApplyGlobalParameters(&argc, argv);
     AZ::Test::printUnusedParametersWarning(argc, argv);
-    AZ::Test::addTestEnvironments({new EditorLibTestEnvironment});
+    AZ::Test::addTestEnvironments({DEFAULT_UNIT_TEST_ENV, new EditorLibTestEnvironment });
```

Comment:
Is the DEFAULT_UNIT_TEST_ENV needed?
The `EditorLibTestEnvironment` should be sufficient for this test

Reply:
is not needed, but the output is annoying and it didnt fail on the assert
`DEFAULT_UNIT_TEST_ENV` has the tracehook

Reply:
Can we just add the `UnitTest::TraceBusHook` as a base of `EditorLibTestEnvironment`?

Reply:
sure